### PR TITLE
style: make the app mobile responsive

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/assets/kf-icons/style.css [hunk 1]
```diff
@@ -14,7 +14,7 @@
 [class^='icon-'],
 [class*=' icon-'] {
   /* use !important to prevent issues with browser extensions that change fonts */
```
Comment (turtlemoji) [inline src/assets/kf-icons/style.css:16]: this is ours or from icomoon?

File: src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -106,6 +106,10 @@ const HomeContainer = styled(Box)({
   margin: '0 auto',
   padding: '15rem 0',
   backgroundColor: 'inherit',
+
+  '@media (max-width: 600px)': {
+    padding: '10.2rem 1.6rem  1rem',
```
Comment (turtlemoji) [inline src/pages/Landing/index.tsx:111]: wrong whitespace makes me crazy

File: src/assets/kf-icons/style.css (hunk 1 above)
Comment (0xArdy) [reply src/assets/kf-icons/style.css:16]: from icomoon

File: src/pages/Landing/index.tsx (hunk 2 above)
Comment (0xArdy) [reply src/pages/Landing/index.tsx]: fixed ser! https://github.com/defi-wonderland/xkeeper-app/pull/19/commits/d5c5c9db6d52f0bb465fad2f073886897f345d89

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**